cmd/utils/nodecmd/accountcmd.go

```diff
@@ -67,6 +67,11 @@ Make sure you backup your keys regularly.`,
 					utils.SingleDBFlag,
```

Comment:
Is it necessary theses options related with DB for `accountcmd`?
@winnie-byun Please check the history and if not please remove all.